screwdriver.yaml

```
@@ -0,0 +1,100 @@
+            - publish-docker: |
+                export DOCKER_TAG=`cat VERSION`
+                ./ci/docker-trigger.sh
+            - save-tag-to-meta: meta set docker_tag $DOCKER_TAG && meta get docker_tag
+        environment:
+            DOCKER_REPO: screwdrivercd/queue-service
+        secrets:
+            # Publishing to NPM
+            - NPM_TOKEN
+            # Pushing tags to Git
+            - GIT_KEY
+            # Trigger a Docker Hub build
+            - DOCKER_TRIGGER
+
+    # Deploy to our beta environment and run tests
+    beta:
+        requires: [publish]
+        steps:
+            - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git ci
+            - wait-docker: DOCKER_TAG=`meta get docker_tag` ./ci/docker-wait.sh
+            - deploy-k8s: K8S_TAG=`meta get docker_tag` ./ci/k8s-deploy.sh
+            #- test: npm install && npm run functional
+        environment:
+            DOCKER_REPO: screwdrivercd/queue-service
+            K8S_CONTAINER: screwdriver-queue-service
+            K8S_IMAGE: screwdrivercd/screwdriver-queue-service
+            K8S_HOST: kubernetes
+            K8S_DEPLOYMENT: sdqueuesvc-beta
+            SD_QUEUE_SVC_HOST: beta.queue.screwdriver.cd
+            K8S_ENV_KEY: DATASTORE_DYNAMODB_PREFIX
+            K8S_ENV_VALUE: beta_rc2_
+            TEST_USERNAME: sd-buildbot
+            TEST_ORG: screwdriver-cd-test
+        secrets:
+            # Access key for functional tests
+            - SD_API_TOKEN
+            # Git access token
+            - GIT_TOKEN
+            # Talking to Kubernetes
+            - K8S_TOKEN
+
+    # Deploy to our prod environment and run tests
+    prod:
+        requires: [beta]
+        steps:
+            - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git ci
+            - wait-docker: DOCKER_TAG=`meta get docker_tag` ./ci/docker-wait.sh
+            - deploy-k8s: K8S_TAG=`meta get docker_tag` ./ci/k8s-deploy.sh
+            # - test: |
```

we need to have this back as well. let it fail in first run and we can add it progressively